js/common/lib/inference-session.ts

```diff
@@ -223,6 +223,8 @@ export declare namespace InferenceSession {
   export interface CudaExecutionProviderOption extends ExecutionProviderOption {
     readonly name: 'cuda';
     deviceId?: number;
+    arenaExtendStrategy?: 0|1;
```

Comment:
use string literal
```ts
arenaExtendStrategy?: 'next-power-of-two' | 'same-as-requested';
```

or, add JSDoc comments explaining their meaning if you want to keep with 0 and 1

Reply:
Ok I mimic-ed some other JSDoc in this file